lore/settings.py

```diff
@@ -104,7 +106,7 @@ def get_var(name, default):
     'guardian.backends.ObjectPermissionBackend',
 )
 
-ROOT_URLCONF = 'ui.urls'
+ROOT_URLCONF = 'lore.urls'
```

Comment:
I guess it makes sense to move the lore.urls back to ui. Not sure if the rest urls should be in the rest app: I do not like having everything in ui, but if this is the path we chose to follow, I guess we have to have everything there.